pdns/dnsdistdist/docs/reference/config.rst

```diff
@@ -349,6 +349,22 @@ Webserver configuration
   * ``custom_headers={[str]=str,...}``: map of string - Allows setting custom headers and removing the defaults.
   * ``acl=newACL``: string - List of IP addresses, as a string, that are allowed to open a connection to the web server. Defaults to "127.0.0.1, ::1".
 
+.. function:: registerWebHandler(path, handler)
+
+  .. versionadded: 1.6.0
+
+  Register a function named ``handler`` that will be called for every query sent to the exact ``path`` path. The function will receive a :class:`WebRequest` object
+  and a :class:`WebResponse` object, representing respectively the HTTP request received and the HTTP response to send.
```

Comment:
could use an example of handling a simple request and setting resp.body etc.

Reply:
That makes total sense, will do!